community/WORKING-GROUPS.md

```diff
@@ -74,6 +75,24 @@ API [resources](../pkg/apis/serving), [validation](../pkg/webhook), and
 | <img width="30px" src="https://github.com/ImJasonH.png"> | Jason Hall | Google  | [ImJasonH](https://github.com/ImJasonH) |
 | <img width="30px" src="https://github.com/mattmoor.png"> | Matt Moore | Google  | [mattmoor](https://github.com/mattmoor) |
 
+## Client
+
+[Client](https://github.com/knative/client), CLI, client libraries, and client conventions
+
+Artifact                   | Link
+-------------------------- | ----
+Forum                      | [knative-dev@](https://groups.google.com/forum/#!forum/knative-dev)
+Community Meeting VC       | [meet.google.com/spi-umby-dmw](https://meet.google.com/spi-umby-dmw) <br>Or dial in:<br>(US) +1 929-445-2235<br>PIN: 824 001#
+Community Meeting Calendar | Tuesdays 10:30a-11:00a Pacific <br>[Calendar Invitation](https://calendar.google.com/event?action=TEMPLATE&tmeid=MjNlNGNscDRmMTAwbDh1ZGZiYWdqMWUya2VfMjAxOTAxMDhUMTgzMDAwWiBnb29nbGUuY29tXzE4dW40ZnVoNnJva3FmOGhtZmZ0bTVvcXE0QGc&tmsrc=google.com_18un4fuh6rokqf8hmfftm5oqq4%40group.calendar.google.com&scp=ALL)
```

Comment:
You might want to switch this to a link to the entire calendar, which won't prompt for Google account sign-in:

```suggestion
Community Meeting Calendar | Tuesdays 10:30a-11:00a Pacific <br>[Calendar Invitation](https://calendar.google.com/calendar/embed?src=google.com_18un4fuh6rokqf8hmfftm5oqq4%40group.calendar.google.com)
```